addons/mail/i18n/mail.pot

```diff
@@ -10712,24 +10693,6 @@ msgid ""
 "continue?"
 msgstr ""
 
-#. module: mail
-#. odoo-python
-#: code:addons/mail/models/discuss/discuss_channel.py:0
-msgid "You are alone in a private conversation."
-msgstr ""
-
-#. module: mail
-#. odoo-python
-#: code:addons/mail/models/discuss/discuss_channel.py:0
-msgid "You are alone in this channel."
-msgstr ""
-
-#. module: mail
-#. odoo-python
-#: code:addons/mail/models/discuss/discuss_channel.py:0
-msgid "You are in a private conversation with %(member_names)s."
-msgstr ""
-
```

Comment:
These strings are still present in `@mail/discuss_channel.py`, I don't get why they are removed.